domain/src/main/assets/welcome.json

```diff
@@ -258,23 +258,30 @@
           }
         ],
         "confirmed_unclassified_answers": [],
-        "customization_args": {
-          "choices": {
-            "value": [
-              "It's translated from a different language.",
-              "It's a nonsense word that someone made up.",
-              "It's the name of a popular cartoon character."
+        "customization_args":{
```

Comment:
@veena14cs This should be reverted back once the PR is approved.

Reply:
okay

Reply:
Per my comment above, it would be better to incorporate this in new explorations dedicated to testing specific scenarios.

Reply:
added new exploration.